draft-sasikumar-id-claim-169.md

```diff
@@ -70,57 +70,228 @@ Claim 169 represents a JSON Object that includes the below table as ID attribute
+  13: US # Nationality
+  14: 2 # Marital status: Married
+  15: Jhon Honai # Guardian
+  16: 03CBABDF83D068ACB5DE65B3CDF25E0036F2C546CB90378C587A076E7A759DFD27CA7872B6CDFF339AEAACA61A6023FD1D305A9B4F33CAA248CEDE38B67D7C915C59A51BB4E77D10077A625258873183F82D65F4C482503A5A01F41DEE612C3542E5370987815E592B8EA2020FD3BDDC747897DB10237EAD179E55B441BC6D8BAD07CE535129CF8D559445CC3A29D746FBF1174DE2E7C0F3439BE7DBEA4520CF88825AAE6B1F291A746AB8177C65B2A459DD19BD32C0C3070004B85C1D63034707CC690AB0BA023350C8337FC6894061EB8A714A8F22FE2365E7A904C72DEC9746ABEA1A3296ECACD1A40450794EDCD2B34844E7C19EB7FB1A4AF3B05C3B374BD2941603F72D3F9A62EAB9A2FDAEEEEC8EE6E350F8A1863C0A0AB1B4058D154559A1CD5133EFCF682ABC339960819C9427889D60380B635A7D21D017974BBA57798490F668ADD86DA58125D9C4C1202CA1308F7734E43E8F77CEB0AF968A8F8B88849F9B98B26620399470ED057E7931DED82876DCA896A30D0031A8CBD7B9EDFDF16C15C6853F4F8D9EEC09317C84EDAE4B349FE54D23D8EC7DC9BB9F69FD7B7B23383B64F22E25F # Binary image
+  17: 2 # Binary image format: JPEG
+  18: [1, 2] # Best quality fingers
+  50: # Right Thumb Biometrics
+
+    # Right Thumb image
+    - 0: 03CBA(...)0378C58 # Data
+      1: 0 # Image
+      2: 1 # JPEG
+
+    # Right Thumb template
+    - 0: 03CBA(...)0378C58 # Data
+      1: 1 # Template
+      2: 100 # Vendor specific
+      3: VendorA # Biometric data issuer
+
+  51: # Right Pointer Finger Biometrics
+
+    # Right Pointer Finger images
+    - 0: 36F2C546(...)CB90378C58 # Data
+      1: 1 # Image
+      2: 6 # WSQ
+      3: VendorA # Biometric data issuer
+
+    - 0: 36F2C546(...)CB90378C58 # Data
+      1: 1 # Image
+      2: 6 # WSQ
+      3: VendorA # Biometric data issuer
+   
+   58: # Left Ring Finger Biometrics   
+
+    # Left Ring Finger templates
+    - 0: 36F2C546(...)CB90378C58 # Data
+      1: 1 # Template
+      2: 1 # Fingerprint Template ISO 19794-2
+      3: VendorA # Biometric data issuer
+      
+    - 0: 36F2C546(...)CB90378C58 # Data
+      1: 1 # Template
+      2: 1 # Fingerprint Template ISO 19794-2
+      3: VendorB # Biometric data issuer
+    
+   60: # Right Iris Biometrics
+
+      # Right Iris images
+    - 0: 36F2C546(...)CB90378C58 # Data
+      1: 1 # Image
+      2: 6 # WSQ
+      3: VendorX # Biometric data issuer
+     
+    - 0: 36F2C546(...)CB90378C58 # Data
+      1: 1 # Image
+      2: 6 # WSQ
+      3: VendorY # Biometric data issuer
+    
+   60: # Left Iris Biometrics
+
+    # Left Iris templates
+    - 0: 36F2C546(...)CB90378C58 # Data
+      1: 1 # Template
+      2: 100 # Vendor specific
+      3: VendorX # Biometric data issuer
+   
+    - 0: 36F2C546(...)CB90378C58 # Data
+      1: 1 # Template
+      2: 100 # Vendor specific
+      3: VendorY # Biometric data issuer
+
+   65: # Voice Biometrics   
+
+    # Voice sound
+    - 0: 03CBA(...)0378C58 # Data
+      2: 1 # MP3
+  
+    # Voice template
+    - 0: 03CBA(...)0378C58 # Data
+      1: 1 # Template
+      2: 100 # Vendor specific
+      3: VendorZ # Biometric data issuer
```

Comment:
There're some whitespace mis-alignments (YAML specific format requires whitespace alignment between siblings), repeated iris field number and add some comments to keep them normalized. 

```suggestion
  50: # Right Thumb Biometrics

    # Right Thumb image
    - 0: 03CBA(...)0378C58 # Data
      1: 0 # Image
      2: 1 # JPEG

    # Right Thumb template
    - 0: 03CBA(...)0378C58 # Data
      1: 1 # Template
      2: 100 # Vendor specific
      3: VendorA # Biometric data issuer

  51: # Right Pointer Finger Biometrics

    # Right Pointer Finger image
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Image
      2: 6 # WSQ
      3: VendorA # Biometric data issuer

    # Right Pointer Finger template
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Template
      2: 1 # Fingerprint Template ISO 19794-2
      3: VendorA # Biometric data issuer

  58: # Left Ring Finger Biometrics

    # Left Ring Finger image
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Image
      2: 6 # WSQ
      3: VendorA # Biometric data issuer

    # Left Ring Finger template
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Template
      2: 1 # Fingerprint Template ISO 19794-2
      3: VendorB # Biometric data issuer

  60: # Right Iris Biometrics

      # Right Iris image
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Image
      2: 6 # WSQ
      3: VendorX # Biometric data issuer

    # Right Iris image
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Image
      2: 6 # WSQ
      3: VendorY # Biometric data issuer

  61: # Left Iris Biometrics

    # Left Iris templates
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Template
      2: 100 # Vendor specific
      3: VendorX # Biometric data issuer
   
    - 0: 36F2C546(...)CB90378C58 # Data
      1: 1 # Template
      2: 100 # Vendor specific
      3: VendorY # Biometric data issuer

  65: # Voice Biometrics

    # Voice sound
    - 0: 03CBA(...)0378C58 # Data
      2: 1 # MP3
  
    # Voice template
    - 0: 03CBA(...)0378C58 # Data
      1: 1 # Template
      2: 100 # Vendor specific
      3: VendorZ # Biometric data issuer
```

Reply:
removed and formatted correctly.